lib/ansible/modules/crypto/openssl_certificate.py

```diff
@@ -543,6 +651,8 @@
 from ansible.module_utils.basic import AnsibleModule, missing_required_lib
 from ansible.module_utils._text import to_native, to_bytes, to_text
 from ansible.module_utils.compat import ipaddress as compat_ipaddress
+from ansible.module_utils.ecs.api import ECSClient, RestOperationException, SessionConfigurationException
+from ansible.module_utils.urls import fetch_url
```

Comment:
I think this is also no longer neccessary.

Reply:
...I am way too used to having an IDE that catches those for me. Thanks.

Reply:
You can always run `flake8` manually on the file to see whether it finds something unnecessary.